packages/walletconnect-v2/src/index.spec.ts

```diff
@@ -28,12 +36,23 @@ const isSwitchEthereumChainRequest = (x: RequestArguments): x is SwitchEthereumC
 }
 
 class MockWalletConnectProvider extends MockEIP1193Provider<number> {
+  opts: EthereumProviderOptions
+
+  constructor(opts: EthereumProviderOptions) {
+    super()
+    this.chainId = opts.chains[0]
+    this.opts = opts
+  }
+
   /** per {@link https://eips.ethereum.org/EIPS/eip-3326#specification EIP-3326} */
-  public eth_switchEthereumChain = jest.fn((args: string) => null)
+  public eth_switchEthereumChain = jest.fn(
+    (/* eslint-disable-line @typescript-eslint/no-unused-vars */ _args: string) => null
```

Comment:
Why didn't this need lint disabling before?

Reply:
I will make this change in a follow-up PR, but TL;DR, ESLint is disabled for test flies in `main`. I have temporarily enabled it locally to format this file with Prettier and applied this comment to ignore the warning. In a follow-up PR, I will re-enable it. 

I could technically do it in a follow-up PR, but like I said, I wanted to run Prettier on it.